### ISSUE TITLE
Panic: Tried to hit test without a DisplayList

### ISSUE THREAD

**Issue by paulrouget** [servo/servo#9845]
```
thread 'ScriptThread PipelineId { namespace_id: PipelineNamespaceId(1), index: PipelineIndex(0) }' panicked at 'Tried to hit test without a DisplayList', /Users/paul/git/servo/components/layout/query.rs:92
note: Run with `RUST_BACKTRACE=1` for a backtrace.
thread 'Constellation' panicked at 'unable to find pipeline - this is a bug', ../src/libcore/option.rs:704
thread 'PaintThread PipelineId { namespace_id: PipelineNamespaceId(1), index: PipelineIndex(0) }' panicked at 'called `Result::unwrap()` on an `Err` value: Error { repr: Custom(Custom { kind: NotFound, error: StringError("Bogus destination port.") }) }', ../src/libcore/result.rs:746
thread 'LayoutThread PipelineId { namespace_id: PipelineNamespaceId(1), index: PipelineIndex(0) }' panicked at 'called `Result::unwrap()` on an `Err` value: IoError(Error { repr: Custom(Custom { kind: Other, error: StringError("Unknown Mach error: 46") }) })', ../src/libcore/result.rs:746
thread '<main>' panicked at 'called `Result::unwrap()` on an `Err` value: "SendError(..)"', ../src/libcore/result.rs:746
thread 'LayoutThread PipelineId { namespace_id: PipelineNamespaceId(0), index: PipelineIndex(0) }' panicked at 'called `Result::unwrap()` on an `Err` value: RecvError', ../src/libcore/result.rs:746
thread 'PaintThread PipelineId { namespace_id: PipelineNamespaceId(0), index: PipelineIndex(0) }' panicked at 'called `Result::unwrap()` on an `Err` value: RecvError', ../src/libcore/result.rs:746
thread 'LayoutWorker worker 1/3' panicked at 'called `Result::unwrap()` on an `Err` value: RecvError', ../src/libcore/result.rs:746
thread 'LayoutWorker worker 2/3' panicked at 'called `Result::unwrap()` on an `Err` value: RecvError', ../src/libcore/result.rs:746
thread 'LayoutWorker worker 3/3' panicked at 'called `Result::unwrap()` on an `Err` value: RecvError', ../src/libcore/result.rs:746
thread 'PaintWorker' panicked at 'called `Result::unwrap()` on an `Err` value: RecvError', ../src/libcore/result.rs:746
thread 'thread 'PaintWorkerPaintWorker' panicked at '' panicked at 'called `Result::unwrap()` on an `Err` value: RecvErrorcalled `Result::unwrap()` on an `Err` value: RecvError', ', ../src/libcore/result.rs../src/libcore/result.rs::746746
```

**Comment by metajack** [comment]
cc @mrobinson @pcwalton

**Comment by paulrouget** [comment]
STR: `servo -w http://firefox.com`, while it's loading, move mouse.

**Comment by paulrouget** [comment]
I was using this PR too: https://github.com/servo/servo/pull/9832
That might be the cause of this issue.

**Comment by mbrubeck** [comment]
I think this was fixed before #9832 landed.  Please re-open if this still happens.
